Cargo.toml

```
@@ -7,7 +7,7 @@ license = "MIT OR Apache-2.0"
 name = "cortex-m-semihosting"
 readme = "README.md"
 repository = "https://github.com/japaric/cortex-m-semihosting"
```

```suggestion
repository = "https://github.com/rust-embedded/cortex-m-semihosting"
```

Can we change this, while we touch the stuff?